apps/browser/src/autofill/background/overlay.background.ts

```diff
@@ -55,7 +61,7 @@ class OverlayBackground implements OverlayBackgroundInterface {
     openAutofillOverlay: () => this.openOverlay(false),
     autofillOverlayElementClosed: ({ message }) => this.overlayElementClosed(message),
     autofillOverlayAddNewVaultItem: ({ message, sender }) => this.addNewVaultItem(message, sender),
-    getAutofillOverlayVisibility: () => this.getOverlayVisibility(),
+    getAutofillOverlayVisibility: async () => await this.getOverlayVisibility(),
```

Comment:
```suggestion
    getAutofillOverlayVisibility: async () => await this.getOverlayVisibility(),
```

Nit: This `async/await` might be unnecessary, though that likely is unclear from the implementation. The implementation within `this.handleExtensionMessage` will handle resolving the promise from `this.getOverlayVisibility` before sending the value back in the `sendResponse` callback. 

The only way this would potentially be necessary is if `this.getOverlayVisibility` is returning a nested promise... 

For example, if `this.getOverlayVisibility` was structured in the following way, it might be necessary (note the missing `await`)

```
private async getOverlayVisibility(): Promise<Promise<InlineMenuVisibilitySetting>> {
  return firstValueFrom(this.autofillSettingsService.inlineMenuVisibility$);
}
```

This would effectively return a promise which on resolution returns a promise, which would then require a resolution. 

Since you're awaiting the `firstValueFrom`, you likely don't need to await the actual result from `this.getOverlayVisibility`.

Reply:
ah yeah; whoops!